Modules/@babylonjs/react-native/shared/BabylonNative.cpp

```diff
@@ -123,7 +124,8 @@ namespace Babylon
                 throw std::runtime_error{ "RenderView can only be called when automatic rendering is disabled." };
             }
 
-            m_graphics->RenderCurrentFrame();
+            if(m_graphics)
+				m_graphics->RenderCurrentFrame();
```

Comment:
>				m_g [](start = 0, length = 7)

nit: we should probably do a pass and make sure all tabs are replaced with spaces

Reply:
the selected text for this comment looks a bit strange. The white space leading this m_graphics->RenderCurrentFrame() uses tabs compared to spaces, which is the actual thing we will want to update